app/src/main/assets/migrations/27.sql

```diff
@@ -1,3 +0,0 @@
-ALTER TABLE TransmitterData ADD COLUMN filtered_data DOUBLE;
```

Comment:
31 is adding it to TransmitterData.  32 is adding it to BgReadings.
According to what I have.  I simply altered 32.sql to remove the creation
of selected_filtered_data to BgReadings, and add in the creation of
filtered_data.


On Sun, May 24, 2015 at 12:33 PM, Stephen Black <notifications@github.com>
wrote:

> In app/src/main/assets/migrations/27.sql
> <https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/30#discussion_r30949694>
> :
>
> > @@ -1,3 +0,0 @@
> > -ALTER TABLE TransmitterData ADD COLUMN filtered_data DOUBLE;
>
> the one in 31 is to add it to the BgReadings table, we also need it in the
> transmitter data table (which is what this particular line Im referencing
> was)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/30/files#r30949694>
> .
>



-- 
John Stevens
"You are how you live, not what you have."